src/www/app/outline_server_repository/access_key_serialization.ts

```diff
@@ -70,7 +70,7 @@ function parseShadowsocksSessionConfigJson(maybeJsonText: string): ShadowsocksSe
 export async function fetchShadowsocksSessionConfig(configLocation: URL): Promise<ShadowsocksSessionConfig> {
   let response;
   try {
-    response = await fetch(configLocation);
+    response = await fetch(configLocation, {cache: 'no-store'});
```

Comment:
This needs a comment. Why is the default behavior inappropriate? It handles stale data by default.

Or perhaps we use `reload`, so we can fallback to the cache (with `only-if-cached `) if the resource is inaccessible?